src/ucs/config/global_opts.c

```diff
@@ -41,7 +42,11 @@ ucs_global_opts_t ucs_global_opts = {
     .stats_format          = UCS_STATS_FULL,
     .rcache_check_pfn      = 0,
     .module_dir            = UCX_MODULE_DIR, /* defined in Makefile.am */
-    .module_log_level      = UCS_LOG_LEVEL_TRACE
+    .module_log_level      = UCS_LOG_LEVEL_TRACE,
+#if defined(__x86_64__)
```

Comment:
@yosefe  These # ifdefs x86 all over common code look awful. I would rather prefer to see arch.h wrapper for this sort of things.